src/test/kotlin/org/rust/ide/refactoring/move/RsMoveTopLevelItemsTest.kt

```diff
@@ -924,6 +924,10 @@ class RsMoveTopLevelItemsTest : RsMoveTopLevelItemsTestBase() {
                 bar::bar_func();
                 let _ = BarStruct {};
             }
+            fn foo2() {
```

Comment:
Lets instead remove imports in `after` code (and mark test with `@UseNewResolve`)